addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.js

```diff
@@ -106,10 +106,7 @@ class ThreadNeedactionPreview extends Component {
             // handled in `_onClickMarkAsRead`
```

Comment:
also event handled system and move full to models? (commit: Move onClickOpen for thread)